AppLovinAdapter/src/main/java/com/chartboost/mediation/applovinadapter/AppLovinAdapter.kt

```diff
@@ -142,6 +142,7 @@ class AppLovinAdapter : PartnerAdapter {
 
     /**
      * Notify AppLovin if the user is underage.
+     * Note: COPPA is no longer supported in the AppLovin SDK 13.0.0
```

Comment:
```suggestion
     * Note: COPPA is no longer supported as of AppLovin SDK 13.0.0
```

Also link to AppLovin docs.